opae.spec.rhel

```diff
@@ -159,6 +145,10 @@ for file in %{buildroot}%{python3_sitelib}/ethernet/{hssicommon,hssiloopback,hss
    chmod a+x $file
 done
 
+# packager
+for file in %{buildroot}%{python3_sitelib}/packager/tools/{afu_json_mgr,packager}.py; do
+   chmod a+x $file
+done
```

Comment:
I think we already have this in the fedora spec, is that right?

Reply:
yes,  packager is already there in fedora spec ,